store/src/java-test/com/zimbra/cs/service/mail/RecoverAccountTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
```

Comment:
don't need this class.

Reply:
removed. also changed enum as per discussion.